cpp/StageInfo.hpp

```diff
@@ -5,9 +5,9 @@
 #ifndef SNOWFLAKECLIENT_STAGEINFO_HPP
 #define SNOWFLAKECLIENT_STAGEINFO_HPP
 
-#include "string"
+#include <string>
 #include <unordered_map>
-#include <snowflake/client.h>
+#include <client_int.h>
```

Comment:
change this to "client_int.h" instead of <client_int.h> to refer the local file specifically. I believe this will cause an error on Windows.